engine/table/src/main/java/io/deephaven/engine/util/input/InputTableUpdater.java

```diff
@@ -21,6 +21,19 @@
  */
 public interface InputTableUpdater {
 
+    /**
+     * Get the input table updater from the given {@code table}, or {@code null} if the input table updater is not set
+     * or is the wrong type.
+     * 
+     * @param table the table
+     * @return the input table updater
+     * @see Table#INPUT_TABLE_ATTRIBUTE
+     */
+    static InputTableUpdater from(Table table) {
+        final Object obj = table.getAttribute(Table.INPUT_TABLE_ATTRIBUTE);
+        return obj instanceof InputTableUpdater ? (InputTableUpdater) obj : null;
```

Comment:
Why silently fail if it is the wrong type?

Reply:
Good point. I think I was originally trying to "play nice" with python, but it's is better to just cast it.